Sitenow footer

adamdelaney opened the pull request:
I've abstracted the most common footer requirements into a module. Please review and provide feedback.

markahrens commented:
Were you using the recently proposed footer designs from OSC for this? I don't know how far they had shared them yet.

![oe-webstyleguide-footer-p4](https://user-images.githubusercontent.com/999218/41937402-cb69afe4-7955-11e8-8461-dfbf5a7658e8.png)

adamdelaney commented:
We could implement this as the default design. The alignment of the social media links and the additional information could be tricky. Also Icons are not supported in the WYSIWYG.

adamdelaney commented:
@markahrens I've added a theme configuration to the form and added a default template file with some css that styles the footer pretty close to the mockup.

adamdelaney commented:
<img width="1207" alt="screen shot 2018-06-27 at 2 34 59 pm" src="https://user-images.githubusercontent.com/1016880/41995418-4c85f0f8-7a17-11e8-9335-18bf6f882887.png">

genjohnson commented:
As a follow-up to the PR, I propose that we update the install profile to enable sitenow_footer and sitenow_wysiwyg_bonus_pack by default: [PR 233](https://github.com/ITS-UofIowa/sitenow_custom_profile/pull/233)

briand44 commented:
The theme configuration isn't showing for me. Uninstalling doesn't clear the saved configuration. The hover text isn't accessible.

adamdelaney commented:
@briand44:
- I've PM you about the theme config stuff.
- I've added hook_unintall() to clean up those variables.
- The hover background color for menu items should probably be handled at the theme layer not from the module.
